src/app/pages/session-detail/session-detail.page.ts

```diff
@@ -117,8 +118,8 @@ export class SessionDetailPage implements OnInit {
     var response = await this.sessionService.getSessionDetailsAPI(this.id);
     var now = moment(response.startDateTime);
     var end = moment(response.endDateTime);
-    var minutes = moment.duration(end.diff(now)).asMinutes();
-    response.duration= minutes+" minutes";
+    var sessionDuration = moment.duration(end.diff(now));
+    response.duration= sessionDuration.hours()==0 ? sessionDuration.minutes()+" Minutes" : sessionDuration.hours()+" Hours "+sessionDuration.minutes()+" Minutes";
```

Comment:
is there any other way to convert it into this format

Reply:
moment was already installed. thats why i used it